packages/expo-google-sign-in/android/src/main/java/expo/modules/google/signin/AuthTask.java

```diff
@@ -0,0 +1,38 @@
+package expo.modules.google.signin;
+
+import expo.core.Promise;
+
+import static expo.modules.google.signin.GoogleSignInModule.ERROR_CONCURRENT_TASK_IN_PROGRESS;
+
+public class AuthTask {
+    private Promise _promise;
```

Comment:
nit but in java the convention we usually follow is `mPromise` and `mTag` rather than `_`